source/index.html

```diff
@@ -1,77 +1,125 @@
-                <ul class="categories">
-                    <li class="category">Clothing Store</li>
-                </ul>
+<body>
+    <section class="head-stone">
+        <nav>
+            <div>
+                <a>Location</a>
+                <a>Shopping</a>
+                <a>Blog</a>
+                <a>Fair Trade</a>
                 <table>
-                    <tr>
-                        <th colspan="3">Hours</th>
-                    </tr>
-                    <tr>
-                        <td>
-                            <abbr title="Monday">M</abbr>
-                            <abbr title="Tuesday">T</abbr>
-                            <abbr title="Wednesday">W</abbr>
-                        </td>
-                        <td>10:30 <abbr title="ante meridiem">am</abbr></td>
-                        <td>6:00 <abbr title="post meridiem">pm</abbr></td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <abbr title="Thursday">Th</abbr>
-                            <abbr title="Friday">F</abbr>
-                        </td>
-                        <td>10:30 <abbr title="ante meridiem">am</abbr></td>
-                        <td>8:00 <abbr title="post meridiem">pm</abbr></td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <abbr title="Saturday">S</abbr>
-                        </td>
-                        <td>11:00 <abbr title="ante meridiem">am</abbr></td>
-                        <td>8:00 <abbr title="post meridiem">pm</abbr></td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <abbr title="Sunday">Su</abbr>
-                        </td>
-                        <td>12:00 <abbr title="post meridiem">pm</abbr></td>
-                        <td>4:00 <abbr title="post meridiem">pm</abbr></td>
-                    </tr>
+                    <tr><img class="social-icon" src="./assets/social/facebook-3-24.png"></tr>
+                    <tr><img class="social-icon" src="./assets/social/instagram-24.png"></tr>
+                    <tr><img class="social-icon" src="./assets/social/pinterest-4-24.png"></tr>
+                    <tr><img class="social-icon" src="./assets/social/yelp-2-24.png"></tr>
                 </table>
             </div>
-            <div class="content-box">
-                <a href="http://goo.gl/maps/7xS1a">
-                    <img src="./images/panara-imports-static-map.png" alt="" width="300" height="300">
-                </a>
+        </nav>
+        <img class="logo" src="./assets/panara-logo.svg" />
+        <h1 class="title">Panara Imports</h1>
+        <h3 class="sub-title">UNIQUE FASHIONS &amp; FABULOUS FINDS SINCE 1992</h3>
+        <p class="email-phone">
+            <a href="mailto:panaramqt@gmail.com">panaramqt@gmail.com</a>
+            <span><a href="tel:906-226-4604">(906) 226-4604</a></span>
```

Comment:
Might want to test this `tel` protocol. I think you had the area code in parenthesis in Poet's, and I think there's probably some rule about how the number should be formatted.